dd-sdk-android-core/api/apiSurface

```diff
@@ -190,6 +190,8 @@ class com.datadog.android.core.SdkReference
   fun get(): com.datadog.android.api.SdkCore?
 fun <T> allowThreadDiskReads(() -> T): T
 fun <T> allowThreadDiskWrites(() -> T): T
+fun StringBuilder.appendIfNotEmpty(String)
+fun StringBuilder.appendIfNotEmpty(Char)
```

Comment:
These functions should be in the `dd-sdk-android-internal` module instead. The problem here is that `core` module is re-exported to the client, so all the symbols here are visible to the client, but we don't want to pollute (make these symbols to appear in the suggestions dropdown) namespace of methods for classes from stdlib, this is only for our usage.

Reply:
moved, thanks for clarifications